packages/wireless-tools/Makefile.patch

```diff
@@ -0,0 +1,25 @@
+diff -uNr wireless_tools.30/Makefile wireless_tools.30.mod/Makefile
+--- wireless_tools.30/Makefile	2009-11-24 20:53:40.000000000 +0200
++++ wireless_tools.30.mod/Makefile	2017-10-17 11:19:33.190969453 +0300
+@@ -5,11 +5,11 @@
+ ## Installation directory. By default, go in /usr/local.
+ ## Distributions should probably use /, but they probably know better...
+ ifndef PREFIX
+-  PREFIX = /usr/local
```

Comment:
You can fix the prefix and cc in `termux_step_post_configure () ` by sed btw,

Reply:
Prefix is set when passing **PREFIX="${TERMUX_PREFIX}"** to make in ```termux_step_make()```. Package builds correctly and everything is placed in ${PREFIX}.

Reply:
@xeffyr have you tried `Wireshark ` on termux?

Reply:
No.